src/webgpu/util/f32_interval.ts

```diff
@@ -98,32 +98,68 @@ function isF32Finite(n: number) {
   return n >= kValue.f32.negative.min && n <= kValue.f32.positive.max;
 }
 
-/** Operation that converts a point to an interval of valid results  */
+/** Function that converts a point to an interval of valid results  */
 export interface PointToInterval {
   (x: number): F32Interval;
 }
 
-/** Operation that converts a pair of points to an interval of valid results  */
+/** Operation used to implement a PointToInterval  */
+interface PointToIntervalOp {
+  /** @returns acceptance interval at x */
+  impl: (x: number) => F32Interval;
+  /** Calculates where in x the min/max extrema of impl occur and returns the span of them.
```

Comment:
It's not clear what the "them" refers to:   The span of the domain values or the range values.
That is, the inputs to impl, or the outputs of impl?

From the usage below, namely `x = op.extrema(x);` I guess it's the *domain* of values.

Reply:
I agree, I will clarify these comments